docs/contributing/feature-flags.md

```diff
@@ -254,10 +254,54 @@ the retrieval methods:
 
 ## Feature flag lifecycle
 
+:::tip LaunchDarkly access
+
 Let your management know when you need to change something about a feature online inside
 LaunchDarkly. Only a small number of users have accounts with LaunchDarkly to save on licensing
 costs.
 
+:::
+
+The process for updating flag values and the ownership of flag changes depends on the environment
+and change as a flag proceeds through our cloud landscape.
+
+### Development environments
+
+In the development environments, ownership of the flag values is in the hands of the development
+team. The Engineering Manager of the team is responsible for maintaining and updating flag values.
+
+By default, flags should be **enabled** in the development environment as soon as the flag is
+created. This allows the feature to be viewed and exercised by Product, Design, the development
```

Comment:
⛏️ I don't consider product or design groups to be proper nouns.

Reply:
Fixed in [`151b341` (#389)](https://github.com/bitwarden/contributing-docs/pull/389/commits/151b341f9f5d78b41d6a78d3315ed79f0659e45d).